packages/core/src/components/text-field/_text-field.scss

```diff
@@ -169,4 +169,10 @@
       }
     }
   }
+
+  textarea[rows].#{$ray-class-prefix}text-area {
```

Comment:
Is the `textarea` part of this selector necessary?

Reply:
probably not. could do `*[rows]` .. but `textarea` is only element that takes in `rows` and none of this really applies to any other html element